app/commands/decidim/system/register_custom_templates.rb

```diff
@@ -30,7 +30,7 @@ def call
           @organization = create_organization
           CreateDefaultPages.call(@organization)
```

Comment:
```suggestion
          CreateCustomDefaultPages.call(@organization)
```
We will need our custom pages based on the template

Reply:
Ok. Done.